auth/build.gradle

```diff
@@ -31,7 +31,7 @@ android {
 dependencies {
     implementation project(path: ':core')
     implementation 'net.openid:appauth'
-
+    implementation 'com.squareup.okhttp3:okhttp:3.9.1'
```

Comment:
should use the http module provided by the core.

Reply:
Forgot to remove this, thanks